WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderActivityLauncher.java

```diff
@@ -255,12 +265,19 @@ private static void openUrlInternal(Context context, @NonNull String url) {
      */
     private static void openUrlExternal(Context context, @NonNull String url) {
         try {
+            // disable deeplinking activity so to not catch WP URLs
+            WPActivityUtils.disableDeepLinking(context);
```

Comment:
This doesn't appear to be working. When I hit the browser icon in the reader detail, it opens the article in a new reader detail instead of the browser.